<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>libraryManageMainWindow</class>
 <widget class="QWidget" name="libraryManageMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome,admin!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/libmanageLogo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_manageBook">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: #f8df72;border:2px groove gray;border-radius:32px;padding:2px 4px;border-style: outset;}
                                           QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
                                           QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_manageUser">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: #619ac3;border:2px groove gray;border-radius:32px;padding:2px 4px;border-style: outset;}
                                           QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
                                           QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancellation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: #495c69;border:2px groove gray;border-radius:32px;padding:2px 4px;border-style: outset;}
                                           QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
                                           QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#f1f0ed;border:2px groove gray;border-radius:33px;padding:2px 4px;border-style: outset;}
                                           QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
                                           QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_hour">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_min">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_sec">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>30</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{background-color:#eeaa9c;border-radius:30px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#eeaa9c;</string>
    </property>
    <property name="text">
     <string>当前注册会员数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numUser">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#eeaa9c</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;7人&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>301</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{background-color:#c4cbcf;border-radius:30px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#c4cbcf;color: #0f1423</string>
    </property>
    <property name="text">
     <string>当前借出书籍:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numBookBorrowed">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#c4cbcf;color: #0f1423</string>
    </property>
    <property name="text">
     <string>&lt;br&gt;1788&lt;/br&gt;本</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#c4cbcf;color: #0f1423</string>
    </property>
    <property name="text">
     <string>库内书籍:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bookInLib">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#c4cbcf;color: #0f1423</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;4758本&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{background-color:#eeaa9c;border-radius:30px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#eeaa9c;</string>
    </property>
    <property name="text">
     <string>管理员总人数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numUser_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#eeaa9c</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;2人&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>501</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Library Manage System</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>51</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房情书体（非商用）</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图
书
管
理
系
统</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_logo</zorder>
  <zorder>pushButton_manageBook</zorder>
  <zorder>pushButton_manageUser</zorder>
  <zorder>pushButton_cancellation</zorder>
  <zorder>pushButton_exit</zorder>
  <zorder>lcdNumber_hour</zorder>
  <zorder>lcdNumber_min</zorder>
  <zorder>lcdNumber_sec</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>label_13</zorder>
  <zorder>label_12</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
